project-progression-policy.md

```diff
@@ -154,6 +154,7 @@ To be considered for Incubation stage, the project must meet the Sandbox require
  * Demonstrate a substantial, in the opinion of the TAC, ongoing flow of commits and merged contributions.
  * Demonstrate that the current level of community participation is sufficient to meet the goals outlined in the growth plan.
  * Since these metrics can vary significantly depending on the type, scope and size of a project, the TAC has final judgement over the level of activity that is adequate to meet these criteria.
+ * Demonstrate that the project is invested in growing a diverse and inclusive community. Resources and recommendations are available in [diversity and inclusion policies](diversity-and-inclusion-policies.md) and through your mentor.
```

Comment:
I don't recall if we discussed this in any TAC meeting, but I am supportive of this additional bullet.